docker-compose.yml

```diff
@@ -12,6 +12,7 @@ networks:
 services:
   elasticsearch:
     build: alegre/elasticsearch
+    platform: linux/x86_64
```

Comment:
Does it build without the `platform` lines?

Reply:
I just built locally without (`docker-compose build elasticsearch`), so I think so! will remove.